src/js/tabs/pid_tuning.js

```diff
@@ -1738,29 +1759,52 @@ pid_tuning.initialize = function (callback) {
             throttleCurve.width = throttleCurve.height *
                 (throttleCurve.clientWidth / throttleCurve.clientHeight);
 
+            const throttleScale = throttleLimitType === THROTTLE_LIMIT_TYPES.SCALE ? throttleLimitPercent : 1;
             const canvasHeight = throttleCurve.height;
             const canvasWidth = throttleCurve.width;
 
             // math magic by englishman
```

Comment:
do we want to keep this comment or append w/ Rosser?

Reply:
I'm an englishman so I say leave it.  :P